src/js/jsx/sections/style/EffectsPanel.jsx

```diff
@@ -34,7 +34,9 @@ define(function (require, exports, module) {
     var os = require("adapter/os"),
         LayerEffect = require("js/models/effects/layereffect"),
         strings = require("i18n!nls/strings"),
+        _ = require("lodash"),
```

Comment:
This isn't your mess, but would you mind tidying these declarations up to match our code style? The typical way to do imports like this is in three different declaration blocks: 1) external libraries (lodash, bluebird, etc.) 2) adapter imports, 3) imports from within spaces-design.